static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.spec.js

```diff
@@ -28,6 +28,7 @@ jest.mock('lib/url', () => ({
 }));
 jest.mock('lib/geolocation', () => ({
     getSync: jest.fn(() => 'GB'),
+    getLocalCurrencySymbol: () => '$',
```

Comment:
oh, maybe this should be `£` since we're mocking to `GB` directly above... could cause confusion